portal-backend/cron/token-prices/src/refresh-prices.js

```diff
@@ -26,18 +26,20 @@ async function fetchPrices() {
 }
 
 const client = redis.createClient(config.get('redis'))
-client.connect()
 
 const expiration = config.get('cacheExpirationMin') * 60
 
-const storePrices = prices =>
-  Promise.all(
+async function storePrices(prices) {
+  client.connect()
+  await Promise.all(
     Object.entries(prices)
       .map(([symbol, price]) =>
         client.set(`price:${symbol}`, price, { EX: expiration }),
       )
       .concat(client.set('time', Date.now())),
   )
+  client.quit() // Release the connection to allow the process to exit
+}
```

Comment:
If there's an error, would the connection still be open?
Perhaps you can use a try / finally and close the connection there.